src/openklant/components/contactmomenten/openapi.yaml

```diff
@@ -448,35 +483,10 @@ paths:
       operationId: audittrail_read
       summary: Een specifieke audit trail regel opvragen.
       description: Een specifieke audit trail regel opvragen.
-      parameters:
-      - name: If-None-Match
-        in: header
-        description: "Voer een voorwaardelijk verzoek uit. Deze header moet \xE9\xE9\
-          n of meerdere ETag-waardes bevatten van resources die de consumer gecached\
-          \ heeft. Indien de waarde van de ETag van de huidige resource voorkomt in\
-          \ deze set, dan antwoordt de provider met een lege HTTP 304 request. Zie\
-          \ [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match)\
-          \ voor meer informatie."
-        required: false
-        examples:
-          oneValue:
-            summary: "E\xE9n ETag-waarde"
-            value: '"79054025255fb1a26e4bc422aef54eb4"'
-          multipleValues:
-            summary: Meerdere ETag-waardes
-            value: '"79054025255fb1a26e4bc422aef54eb4", "e4d909c290d0fb1ca068ffaddf22cbd0"'
-        schema:
-          type: string
       responses:
         '200':
           description: OK
           headers:
-            ETag:
-              description: De ETag berekend op de response body JSON. Indien twee
-                resources exact dezelfde ETag hebben, dan zijn deze resources identiek
-                aan elkaar. Je kan de ETag gebruiken om caching te implementeren.
-              schema:
-                type: string
```

Comment:
@stevenbal I need some advice here:

I regenerated the OAS after updating the dependencies and somehow the ETag support information disappeared. 

I traced this back to updating `vng-api-common` from 1.6.4 to 1.9.0, and starting from 1.7.3 this was changed: https://github.com/VNG-Realisatie/vng-api-common/releases/tag/1.7.3

So I was wondering if Etag support ever was a supported or working feature, or if it just showed up in the OAS because of an issue that was fixed from 1.7.3 (and a few later ones), like I can't find any use of the expected related mixins or decorators on the views and models etc in open-klant.

Reply:
I vaguely remember that the ETag headers in the OAS were initially regardless of whether the associated resources actually supported caching (indeed the Open Klant resources didn't support this) and that this behavior was indeed changed in 1.7.3.

So I think it's fine to drop these headers :+1: